toolkit/tools/imagecustomizer/docs/samples/deploy-pxe-artifacts.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
```

Comment:
I agree it's not doing something fancy. Sometimes minor details, like permissions on files, can save people some time.

I've removed it.

#1, it's really up to the person doing the deployment since they may choose to have an http server or something else to server the iso file.
#2, was left in there by mistake.